webservices/rest.py

```diff
@@ -219,6 +219,16 @@ def add_aggregate_resource(api, view, schedule, label):
     '/committee/<committee_id>/filings/',
     '/candidate/<candidate_id>/filings/',
 )
+
+api.add_resource(
+    filings.EFilingSummaryView,
+    '/efile/presidential/<string:form_type>/',
```

Comment:
Small note, my preference for the url would be `efile/reports/presidential` feel free to push back if that would be more cumbersome.